### PR TITLE
Add Onlyoffice.com.xml

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

**Comment by pipboy96** [comment]
There are more subdomains:
```
ascendant.onlyoffice.com - Found open ports: 443, 80
api-system.onlyoffice.com - Found open ports: 443, 80
all.onlyoffice.com - Found open ports: 443, 80
onlyoffice.com - Found open ports: 443, 80
anaidian.onlyoffice.com - Found open ports: 443, 80
apusiaem.onlyoffice.com - Found open ports: 443, 80
agenciapr.onlyoffice.com - Found open ports: 443, 80
1plugin.onlyoffice.com - Found open ports: 443, 80
acertius.onlyoffice.com - Found open ports: 443, 80
www.api.onlyoffice.com - Found open ports: 443, 80
1-demo.onlyoffice.com - Found open ports: 443, 80
armrecouvrement.onlyoffice.com - Found open ports: 443, 80
aacsdocuments.onlyoffice.com - Found open ports: 443, 80
api.onlyoffice.com - Found open ports: 443, 80
apps.onlyoffice.com - Found open ports: 443, 80
payments.onlyoffice.com - Found open ports: 443, 80
auth.onlyoffice.com - Found open ports: 443, 80
www.onlyoffice.com - Found open ports: 443, 80
cambridgejmb.onlyoffice.com - Found open ports: 443, 80
cdobyouthministry.onlyoffice.com - Found open ports: 443, 80
buildserver.onlyoffice.com - Found open ports: 443, 80
ccusersforum.onlyoffice.com - Found open ports: 443, 80
centrify.onlyoffice.com - Found open ports: 443, 80
controlpanel.onlyoffice.com - Found open ports: 443, 80
capufaktur.onlyoffice.com - Found open ports: 443, 80
download.onlyoffice.com - Found open ports: 443, 80
ctfbay.onlyoffice.com - Found open ports: 443, 80
www.demodocs.onlyoffice.com - Found open ports: 443, 80
converter.onlyoffice.com - Found open ports: 443, 80
c3prevention.onlyoffice.com - Found open ports: 443, 80
feedback.onlyoffice.com - Found open ports: 443, 80
design.onlyoffice.com - Found open ports: 443, 80
doc.onlyoffice.com - Found open ports: 443, 80
ergowerk.onlyoffice.com - Found open ports: 443, 80
bugzilla.onlyoffice.com - Found open ports: 443, 80
etherpad.onlyoffice.com - Found open ports: 443, 80
entell6.onlyoffice.com - Found open ports: 443, 80
exablate.onlyoffice.com - Found open ports: 443, 80
exception.onlyoffice.com - Found open ports: 443, 80
finalix.onlyoffice.com - Found open ports: 443, 80
affiliates.onlyoffice.com - Found open ports: 443, 80
finnovaregio.onlyoffice.com - Found open ports: 443, 80
doc-v4.1-demo.onlyoffice.com - Found open ports: 443, 80
forum.onlyoffice.com - Found open ports: 443, 80
cdn.onlyoffice.com - Found open ports: 443, 80
helpdesk.onlyoffice.com - Found open ports: 443, 80
frontlinemanagementllc.onlyoffice.com - Found open ports: 443, 80
gehao1.onlyoffice.com - Found open ports: 443, 80
get.onlyoffice.com - Found open ports: 443, 80
goihlux.onlyoffice.com - Found open ports: 443, 80
gappmu.onlyoffice.com - Found open ports: 443, 80
gonfib.onlyoffice.com - Found open ports: 443, 80
hartsmill.onlyoffice.com - Found open ports: 443, 80
help.onlyoffice.com - Found open ports: 443, 80
cryptoguard.onlyoffice.com - Found open ports: 443, 80
ilight.onlyoffice.com - Found open ports: 443, 80
hejing.onlyoffice.com - Found open ports: 443, 80
helpcenter.onlyoffice.com - Found open ports: 443, 80
hoeksteenamersfoort.onlyoffice.com - Found open ports: 443, 80
innovaweblab.onlyoffice.com - Found open ports: 443, 80
intranetlek.onlyoffice.com - Found open ports: 443, 80
intranet.onlyoffice.com - Found open ports: 443, 80
it.onlyoffice.com - Found open ports: 443, 80
intelika.onlyoffice.com - Found open ports: 443, 80
jaziegrl.onlyoffice.com - Found open ports: 443, 80
lerhlh.onlyoffice.com - Found open ports: 443, 80
doc-v4-plugins-demo.onlyoffice.com - Found open ports: 443, 80
enactus.onlyoffice.com - Found open ports: 443, 80
demodocs.onlyoffice.com - Found open ports: 443, 80
envreview.onlyoffice.com - Found open ports: 443, 80
lifechangecenters.onlyoffice.com - Found open ports: 443, 80
license.onlyoffice.com - Found open ports: 443, 80
luxsystems.onlyoffice.com - Found open ports: 443, 80
lucy-home.onlyoffice.com - Found open ports: 443, 80
maklernetz.onlyoffice.com - Found open ports: 443, 80
mta-sts.mail.onlyoffice.com - Found open ports: 443, 80
daimler.onlyoffice.com - Found open ports: 443, 80
microproxy.onlyoffice.com - Found open ports: 443, 80
minsas2015.onlyoffice.com - Found open ports: 443, 80
www.mail.onlyoffice.com - Found open ports: 443, 80
multitraslados.onlyoffice.com - Found open ports: 443, 80
mx1.onlyoffice.com - Found open ports: 443, 80
marxandlieberman.onlyoffice.com - Found open ports: 443, 80
myhomes.onlyoffice.com - Found open ports: 443, 80
www.mx1.onlyoffice.com - Found open ports: 443, 80
nctautotest-280733869-documents-documents-sharing.onlyoffice.com - Found open ports: 443, 80
nctautotest-462523073-smoke-documents-module-shari.onlyoffice.com - Found open ports: 443, 80
mta-sts.mx1.onlyoffice.com - Found open ports: 443, 80
nctautotest-923613556-documents-sharing-for-accoun.onlyoffice.com - Found open ports: 443, 80
nctautotest-564872715-smoke-documents-module-shari.onlyoffice.com - Found open ports: 443, 80
nctautotest-531637950-documents-documents-sharing.onlyoffice.com - Found open ports: 443, 80
nordicdefenseestates.onlyoffice.com - Found open ports: 443, 80
normanonline.onlyoffice.com - Found open ports: 443, 80
null-state.onlyoffice.com - Found open ports: 443, 80
office.onlyoffice.com - Found open ports: 443, 80
one-click-install.onlyoffice.com - Found open ports: 443, 80
www.owncloud.onlyoffice.com - Found open ports: 443, 80
partners.onlyoffice.com - Found open ports: 443, 80
billing.onlyoffice.com - Found open ports: 443, 80
personal.onlyoffice.com - Found open ports: 443, 80
www.nextcloud.onlyoffice.com - Found open ports: 443, 80
phifactor.onlyoffice.com - Found open ports: 443, 80
pkdocs.onlyoffice.com - Found open ports: 443, 80
plexicoat.onlyoffice.com - Found open ports: 443, 80
www.pkdocs.onlyoffice.com - Found open ports: 443, 80
pk2017.onlyoffice.com - Found open ports: 443, 80
portal-eds.onlyoffice.com - Found open ports: 443, 80
pojawarehouse.onlyoffice.com - Found open ports: 443, 80
rasdelivers.onlyoffice.com - Found open ports: 443, 80
possunt.onlyoffice.com - Found open ports: 443, 80
reaa1802roussillon.onlyoffice.com - Found open ports: 443, 80
static.onlyoffice.com - Found open ports: 443, 80
rhsppsite.onlyoffice.com - Found open ports: 443, 80
reg.onlyoffice.com - Found open ports: 443, 80
projectcce.onlyoffice.com - Found open ports: 443, 80
rreese568.onlyoffice.com - Found open ports: 443, 80
rotseepark.onlyoffice.com - Found open ports: 443, 80
service.onlyoffice.com - Found open ports: 443, 80
scmgenterprises.onlyoffice.com - Found open ports: 443, 80
slaterglobal.onlyoffice.com - Found open ports: 443, 80
streamlinekids.onlyoffice.com - Found open ports: 443, 80
spam.onlyoffice.com - Found open ports: 443, 80
www.store.onlyoffice.com - Found open ports: 443, 80
slamzwl.onlyoffice.com - Found open ports: 443, 80
tchangers.onlyoffice.com - Found open ports: 443, 80
temstms.onlyoffice.com - Found open ports: 443, 80
schemas.onlyoffice.com - Found open ports: 443, 80
thegaintrust.onlyoffice.com - Found open ports: 443, 80
thumb.onlyoffice.com - Found open ports: 443, 80
theguild.onlyoffice.com - Found open ports: 443, 80
translate.onlyoffice.com - Found open ports: 443, 80
static-www.onlyoffice.com - Found open ports: 443, 80
support.onlyoffice.com - Found open ports: 443, 80
nextcloud.onlyoffice.com - Found open ports: 443, 80
track.onlyoffice.com - Found open ports: 443, 80
nct.onlyoffice.com - Found open ports: 443, 80
uploads.onlyoffice.com - Found open ports: 443, 80
owncloud.onlyoffice.com - Found open ports: 443, 80
van-it.onlyoffice.com - Found open ports: 443, 80
trikuharri.onlyoffice.com - Found open ports: 443, 80
watdev.onlyoffice.com - Found open ports: 443, 80
virtualmediaagency360.onlyoffice.com - Found open ports: 443, 80
write616.onlyoffice.com - Found open ports: 443, 80
watertec.onlyoffice.com - Found open ports: 443, 80
store.onlyoffice.com - Found open ports: 443, 80
test.onlyoffice.com - Found open ports: 443, 80
testdevice.onlyoffice.com - Found open ports: 443, 80
univention.onlyoffice.com - Found open ports: 443, 80
```

**Comment by pipboy96** [comment]
Actually, there is a wildcard DNS record.

**Comment by pipboy96** [comment]
I'm not sure what to do, since enumerating all subdomains is impossible because of both wildcard DNS record and certificate.

**Comment by J0WI** [comment]
I've added a hint.

File: src/chrome/content/rules/Onlyoffice.com.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!--
+	Host has wildcard DNS and cert.
+-->
+<ruleset name="Onlyoffice.com">
```
Comment (pipboy96) [inline src/chrome/content/rules/Onlyoffice.com.xml]:
```suggestion
<ruleset name="Onlyoffice.com (partial)">
```

**Comment by pipboy96** [comment]
@J0WI There is also a `.org` variant of this domain, also with a wildcard record.

File: src/chrome/content/rules/Onlyoffice.com.xml (hunk 1 above)
Comment (J0WI) [reply src/chrome/content/rules/Onlyoffice.com.xml]: I think we covered all common hosts.

Comment (pipboy96) [reply src/chrome/content/rules/Onlyoffice.com.xml]: Well, there's no way to get all hosts, so I would probably mark as `(partial)`.

**Comment by ivysrono** [comment]
https://travis-ci.org/EFForg/https-everywhere/jobs/586533856#L327
```ERROR src/chrome/content/rules/Onlyoffice.org.xml: Fetch error: 

 => 

 not enough values to unpack (expected 2, got 1)

ERROR src/chrome/content/rules/Onlyoffice.org.xml: Fetch error: 

 => 

 not enough values to unpack (expected 2, got 1)```

File: src/chrome/content/rules/Onlyoffice.org.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	For other Onlyoffice coverage, see Onlyoffice.com.xml.
+
+	Expired:
+		- onlyoffice.org
+		- www.onlyoffice.org
+
+	Host has wildcard DNS.
+-->
+<ruleset name="Onlyoffice.org">
+	<target host="onlyoffice.org" />
```
Comment (ivysrono) [inline src/chrome/content/rules/Onlyoffice.org.xml]: https://travis-ci.org/EFForg/https-everywhere/jobs/586533856#L327

Comment (J0WI) [reply src/chrome/content/rules/Onlyoffice.org.xml]: Let's see if this happens again...

Comment (ivysrono) [reply src/chrome/content/rules/Onlyoffice.org.xml]: https://travis-ci.org/EFForg/https-everywhere/jobs/586653806#L327
again

Comment (ivysrono) [reply src/chrome/content/rules/Onlyoffice.org.xml]: https://travis-ci.org/EFForg/https-everywhere/jobs/592913801#L334
again and again.

Comment (pipboy96) [reply src/chrome/content/rules/Onlyoffice.org.xml]: I guess remove these targets, mark them as "SSL error" and mark the ruleset as "partial".

File: src/chrome/content/rules/Onlyoffice.org.xml [hunk 3]
```diff
@@ -13,6 +13,6 @@
 	<target host="dev.onlyoffice.org" />
 	<target host="cloud.onlyoffice.org" />
 
-	<rule from="^http://(www\.)?onlyoffice\.org/" to="https://$1onlyoffice.com/" />
+	<rule from="^http://(www\.)?onlyoffice\.org/" to="https://www.onlyoffice.com/" />
```
Comment (pipboy96) [inline src/chrome/content/rules/Onlyoffice.org.xml]:
```
	<rule from="^http://onlyoffice\.org/" to="https://www.onlyoffice.com/" />
```

Comment (ivysrono) [reply src/chrome/content/rules/Onlyoffice.org.xml]: https://travis-ci.org/EFForg/https-everywhere/jobs/593026985#L335

Comment (pipboy96) [reply src/chrome/content/rules/Onlyoffice.org.xml]: oops

Comment (J0WI) [reply src/chrome/content/rules/Onlyoffice.org.xml]: Let's just try to whitelist this

**Pull request merged**